packages/backend-common/src/deprecated/index.ts

```diff
@@ -71,6 +85,55 @@ export type CacheClientSetOptions = CacheServiceSetOptions;
  */
 export type CacheClientOptions = CacheServiceOptions;
 
+/**
+ * @public
+ * @deprecated Use `DatabaseManager` from the `@backstage/backend-defaults` package instead
+ */
+export class DatabaseManager implements LegacyRootDatabaseService {
```

Comment:
> [!Note
> I could not extends the `_DatabaseManager` class as its constructor is private and could not assign it to constant because constants cannot be used as a class type.

Reply:
I start getting the error of constants cannot be used as type:
![image](https://github.com/backstage/backstage/assets/6290749/97a1754a-9fa7-4295-a0b4-f6dfee14fbb7)

Reply:
I believe the erro only happens to classes because they can be used both as value and type...